src/kudu/tserver/simple_tablet_manager.h

```diff
@@ -122,13 +130,27 @@ class TSTabletManager : public consensus::ConsensusRoundHandler {
     state_ = s;
   }
 
+  Status WaitUntilRunning();
+
+  Status WaitUntilConsensusRunning(const MonoDelta& timeout);
+
+  Status CreateNew(FsManager *fs_manager);
+
+  Status SetupRaft();
+
+  Status InitRaftAsync(bool is_first_run);
```

Comment:
nit: Most of them seem pretty straightforward after reading the cc file. Maybe leave one-liners in the headers to briefly explain what the member functions do. I have this habit of reading only headers. Upto you though.

Reply:
Yes, we should document these functions.